vnext/codegen/NativeImageLoaderAndroidSpec.g.h

```diff
@@ -13,12 +13,28 @@
 
 namespace Microsoft::ReactNativeSpecs {
 
+REACT_STRUCT(ImageLoaderAndroidSpec_getSize_returnType)
+struct ImageLoaderAndroidSpec_getSize_returnType {
```

Comment:
>ImageLoaderAndroidSpec_getSize_returnType [](http://example.com/codeflow?start=7&length=41)

Not for this PR, but it would be great to generate these structs inside of the "struct ImageLoaderAndroidSpec", and also have a shorter name. e.g. getSize_result .
Though, in this specific case the ideal name would be "Size" and we can have it for both functions.

Reply:
Ideally people could create a type alias or interface (not supported yet but I am going to do it), and they can control how names are generated.